scss/_variables.scss

```diff
@@ -1,34 +1,94 @@
+$spacer: 6px !default;
+$spacers: (
+  0,
+  round($spacer / 2),
+  $spacer,
+  $spacer * 2,
+  $spacer * 4,
+  $spacer * 6
+) !default;
+
+// Aliases for easy use
+$spacer1: nth($spacers, 2) !default; // 3px
+$spacer2: nth($spacers, 3) !default; // 6px
+$spacer3: nth($spacers, 4) !default; // 12px
+$spacer4: nth($spacers, 5) !default; // 24px
+$spacer5: nth($spacers, 6) !default; // 36px
+
 $container-width: 980px !default;
 $grid-gutter:     10px !default;
 
-// Brand colors
-$brand-blue:       #4078c0 !default;
-$brand-gray-light: #999    !default;
-$brand-gray:       #767676 !default;
-$brand-gray-dark:  #333    !default;
-$brand-green:      #6cc644 !default;
-$brand-red:        #bd2c00 !default;
-$brand-orange:     #c9510c !default;
-$brand-purple:     #6e5494 !default;
-
-// Font stack
-$body-font: Helvetica, arial, nimbussansl, liberationsans, freesans, clean, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
-
-// The base body size
-$body-font-size: 13px !default;
+// ---- Colors ----
 
-// Monospace font stack
-$mono-font: Consolas, "Liberation Mono", Menlo, Courier, monospace !default;
+// Brand
+$blue:              #4078c0 !default;
+$gray-dark:         #333    !default;
+$gray-light:        #999    !default;
+$gray:              #767676 !default;
+$green:             #6cc644 !default;
+$orange:            #c9510c !default;
+$purple:            #6e5494 !default;
+$red:               #bd2c00 !default;
+$white:             #fff !default;
 
 // State indicators.
-$status-open:     $brand-green;
-$status-closed:   $brand-red;
-$status-reverted: $status-closed;
-$status-merged:   $brand-purple;
-$status-renamed:  #fffa5d;
-$status-pending:  #cea61b;
+$status-renamed:    #fffa5d !default;
+$status-pending:    #cea61b !default;
 
 // Repository type colors
-$repo-private-text: #a1882b;
-$repo-private-bg:   #fff9ea;
-$repo-private-icon: #e9dba5;
+$repo-private-text: #a1882b !default;
+$repo-private-bg:   #fff9ea !default;
+$repo-private-icon: #e9dba5 !default;
+
+// Border colors
+$border-blue:       #c5d5dd !default;
+$border-gray-dark:  #ddd !default;
+$border-gray-light: #eee !default;
+$border-gray:       #e5e5e5 !default;
+
+// Background colors
+$bg-blue-light:     #f2f8fa !default;
+$bg-blue:           $blue !default;
+$bg-gray-dark:      $gray-dark !default;
+$bg-gray-light:     #fafafa !default;
+$bg-gray:           #f5f5f5 !default;
+$bg-green:          $green !default;
+$bg-orange:         $orange !default;
+$bg-purple:         $purple !default;
+$bg-red:            $red !default;
+$bg-white:          $white !default;
+
+// Text colors
+$text-blue:         $blue !default;
+$text-gray-dark:    $gray-dark !default;
+$text-gray-light:   $gray-light !default;
+$text-gray:         $gray !default;
+$text-green:        darken($green, 10%) !default;
+$text-orange:       $orange !default;
+$text-purple:       $purple !default;
+$text-red:          $red !default;
+$text-white:        $white !default;
```

Comment:
Some of these `$text-` and `$bg-` variables are the same, but I wanted to make separate variables so that it could set us up for future a11y tests. For example the `$text-green` variable was darkend 10% because it was more readable.